pom.xml

```diff
@@ -124,7 +124,7 @@
         <orc-core.version>1.8.3</orc-core.version>
         <awssdk.version>2.24.12</awssdk.version>
         <aliyun-sdk-oss.version>3.10.2</aliyun-sdk-oss.version>
-        <terminal.spark.version>3.3.2</terminal.spark.version>
+        <terminal.spark.version>3.3.4</terminal.spark.version>
```

Comment:
Is the upgrade to 3.3.4 here because 3.3.2 triggers a bug when using paimon?

Reply:
If use 3.3.2,couldn't avliable ci.paimon some bug fixed in this version,1.1.1 dependency 3.2.4